UserGuide.md

```diff
@@ -186,7 +189,7 @@ Write the title on the first line of the question document. To annotate the tile
 2.	Write the title of the lesson using headline capitalization. 
```

Comment:
On line 187, `... To annotate the tile...` is this `tile` or `title`? Sorry I can not comment on the above line. Thanks.

Reply:
<code>title</code> thanks for pointing that out.